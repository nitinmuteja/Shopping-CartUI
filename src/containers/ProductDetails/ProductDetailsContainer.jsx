import React from 'react';
import ProductInformation from "../../components/ProductInformation";

class ProductDetailsContainer extends React.Component {
    render() {
        return (<div>
            <ProductInformation/>

        </div>)


    }

}

export default ProductDetailsContainer;