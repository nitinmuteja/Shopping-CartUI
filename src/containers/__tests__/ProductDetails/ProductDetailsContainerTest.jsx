import React from 'react';
import { render } from '@testing-library/react';

import ProductDetailsContainer from "../../ProductDetails/ProductDetailsContainer";


jest.mock('../../../components/ProductInformation',()=>()=>(<div>Dummy Product Information Component</div>));

describe('Product Details Component', () => {


    it('should match snapshot', () => {
        const { container } = render(<ProductDetailsContainer />);

        expect(container).toMatchSnapshot();
    });
});
