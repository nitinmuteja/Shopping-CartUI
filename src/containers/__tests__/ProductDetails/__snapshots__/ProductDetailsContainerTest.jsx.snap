// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Details Component should match snapshot 1`] = `
<div>
  <div>
    <div>
      Dummy Product Information Component
    </div>
  </div>
</div>
`;
